components/molecules/ContributorProfileInfo/contributor-profile-info.tsx

```diff
@@ -54,7 +54,7 @@ const ContributorProfileInfo = ({
           {isConnected && <Badge text="beta" />}
         </div>
         <div className="flex items-center text-sm gap-3">
-          <span className="text-light-slate-11 text-sm">{`@${githubName}`}</span>
+          <a className="text-light-slate-11 text-sm" href={`https://github.com/${githubName}`}>{`@${githubName}`}</a>
```

Comment:
```suggestion
          <span className="text-light-slate-11 text-sm">{`@${githubName}`}</span>
```
I don't think the `@[username]` should link out to GitHub, because visually it doesn't look like something that should link to GitHub, I think its best for User Experience to either have this just link back to the same profile on open-sauced/insights or stay without a link at all.

The `GitHub` button link in the Socials Section below the About section, unlike the `@[username]` tells me exactly what/where I'm expecting to land when I click, and its feels just enough.

Reply:
Why don't we use the `Link` component instead of using `a` directly

Reply:
This was done on purpose so we can discuss the better one that fits our needs. The GitHub link below the about section pretty much fits our needs here. So we can move on with this 👍

Reply:
> Why don't we use the `Link` component instead of using `a` directly

The `Link` tag does not support external URLs in a new tab

Reply:
Discard my previous comment... Just found out that links can be used in this case since Next13 https://stackoverflow.com/a/74382873 :<

Reply:
I might have a counter point similar to @babblebey. I might argue that the `@[username]` is not needed. The GitHub profile being linkable below is fine and we are not using real names on the platform so the username is already repeated multiple places.

Reply:
would this be a better design: 
![image](https://github.com/open-sauced/insights/assets/18273833/8ef71500-2453-4d33-8762-89e4ad7a86c4)

Reply:
@babblebey, @NsdHSO thank you for your feedback!

Reply:
> would this be a better design: 
> 
> ![image](https://github.com/open-sauced/insights/assets/18273833/8ef71500-2453-4d33-8762-89e4ad7a86c4)
> 
> 

This is much better me

Reply:
This is the current implementation, 
![image](https://github.com/open-sauced/insights/assets/18273833/6a878f62-7ae3-469d-ae05-8baf63dccaa0)

Side note, I noticed the image you added after writing my comment, the page did not load the edit.